src/data/nat/modeq.lean

```diff
@@ -209,6 +209,13 @@ lemma add_div_eq_of_add_mod_lt {a b c : ℕ} (hc : a % c + b % c < c) :
 if hc0 : c = 0 then by simp [hc0]
 else by rw [add_div (nat.pos_of_ne_zero hc0), if_neg (not_le_of_lt hc), add_zero]
 
+lemma add_div_eq_of_dvd {a b c : ℕ} (hca : c ∣ a) :
+  (a + b)/c = a/c + b/c :=
```

Comment:
Do you mind adding the `left` variant as well?
```suggestion
protected lemma add_div_of_dvd_right {a b c : ℕ} (hca : c ∣ a) :
  (a + b) / c = a / c + b / c :=
```
I removed the `_eq` to make it consistent with the `int` version (and added `protected` so that there won't be confusion between them). This does makes this name inconsistent with the surrounding `nat` lemmas, but I think that's less of a big deal.